tester_app/client/bootstrap-sass.config.js

```diff
@@ -0,0 +1,88 @@
+// IMPORTANT: Make sure to keep the customizations defined in this file
+//            in-sync with the ones defined in app/assets/stylesheets/_bootstrap-custom.scss.
+
```

Comment:
// See https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/_bootstrap.scss for reference